### ISSUE TITLE
Regression on Travis

### ISSUE THREAD

**Issue by lsolesen** [vih-elevforening/elevforeningen-build#23]
This build caused the tests to fail: https://travis-ci.org/vih-elevforening/elevforeningen-build/builds/65019268

**Comment by mglaman** [comment]
Yeah, we need to find a way to add the test users to a group before proceeding.

**Comment by mglaman** [comment]
My latest commit should fix this, however http://files.aegir.cc/core/drupal-7.38.1.tar.gz. is throwing 404 preventing Travis from doing its magic.
